File: programme/degree-to-radian/degree-to-radian.js
// Write a programme to convert degree to radian

function degrees_to_radians(degrees) {
  var pi = Math.PI;
  return degrees * (pi / 180);
}

degree = 1;
console.log("Input  : " + degree);
console.log("Output : " + degrees_to_radians(degree).toFixed(5));

File: programme/degree-to-radian/degree-to-radian.cs
// Write a programme to convert degree to radian
using System;

public class DegreeToRadian
{
    static void Main(string[] args)
    {
        Console.Write("Input degrees : ");

        double degrees;
        while (!double.TryParse(Console.ReadLine(), out degrees))
        Console.WriteLine("Invalid Input!!");

        double radians = degrees * Math.PI / 180;
        Console
            .WriteLine($"\n{degrees} Degrees is {Math.Round(radians, 5)} Radians.");
    }
}
